modules/local/dream/differential/main.nf

```diff
@@ -0,0 +1,81 @@
+// TODO:
+//      should this module take all variables already processed (meta),
+//      or should it be able to take the yaml file and parse it on its own?
+
+process DREAM_DIFFERENTIAL {
+    tag "${meta.id}"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml" // TODO: COMPLETE THIS TOO!
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/d6/d6fa8a7908dd357484d63bb574a378f1739440a2c0752ce91b1e0e9d1ac1638c/data' :
+        'community.wave.seqera.io/library/bioconductor-edger_bioconductor-variancepartition_r-optparse:ba778938d72f30c5' }"
+
+    input:
+    val meta                                                // array with id, variable, reference, target, blocking_factors, and more in the future.
```

Comment:
Why are you passing only the meta? It looks like it would make more sense to have a single input chanel, with only one meta, and to pass the meta there. i.e:
tuple val(meta), path(samplesheet), path(intensities)